Pipfile

```diff
@@ -17,6 +17,7 @@ sqlalchemy-stubs = "*"
 flask = "~=1.1"
```

Comment:
Can you run `pipenv update` too since we haven't touched this project in a while.